python-frontend/src/main/java/org/sonar/plugins/python/api/PythonVersionUtils.java

```diff
@@ -106,7 +106,9 @@ private PythonVersionUtils() {
   }
 
   public static Set<Version> fromString(String propertyValue) {
-    String[] versions = propertyValue.split(",");
+    return fromString(propertyValue.split(","));
+  }
+  public static Set<Version> fromString(String[] versions) {
```

Comment:
Would it make sense to rename this to `fromArray`, `fromVersionArray`, `fromStringArray` or anything of the kind that would be more accurate? I find it weird that a method called `fromString` actually takes an array as parameter.